packages/react-components/react-nav-preview/src/components/NavCategoryItem/renderNavCategoryItem.tsx

```diff
@@ -18,13 +18,9 @@ export const renderNavCategoryItem_unstable = (
   return (
     <NavCategoryItemProvider value={contextValues.navCategoryItem}>
       <state.root>
-        {/* TODO: These were copied from AccordionHeader.
-        We need to decide if they are still applicable
-        in our scenario and how to adapt them. */}
-        {/* <state.button> */}
-        {/* Todo: {state.icon && <state.icon />} */}
+        {state.icon && <state.icon />}
         {state.root.children}
-        {state.expandIcon && <state.expandIcon />} {/* </state.button> */}
+        {state.expandIcon && <state.expandIcon />}
```

Comment:
If you take the comment about the position this would have to be:
```ts
{state.position === 'before' && state.icon && <state.icon />}
{state.root.children}
{state.position === 'after' && state.icon && <state.icon />}
```

Reply:
Recapping offline discussion - we're not going to have a position prop for the time being.